lib/Elastica/Cluster.php

```diff
@@ -73,14 +73,7 @@ public function getResponse()
      */
     public function getIndexNames()
     {
-        $metaData = $this->_data['metadata']['indices'];
-
-        $indices = array();
-        foreach ($metaData as $key => $value) {
-            $indices[] = $key;
-        }
-
-        return $indices;
+        return array_keys($this->_data['metadata']['indices']);
```

Comment:
Looks to me like it does the same. To we have a unit test that covers this? You can easily check in the coverage report.